src/BBT.MaybePattern.Tests/BBT.MaybePattern.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I find it strange to find the test project in .NET6 while the project only addresses .NET Standard 2.0. I would expect the main project to also support .NET 6. If there are technical reasons for this, it is ok.

Reply:
The main project also references now .NET 6

Reply:
Now its multitargeting, this is because of an internal reference we have